src/main/java/com/github/dockerjava/api/model/PushResponseItem.java

```diff
@@ -18,7 +18,7 @@
      */
     @JsonIgnore
     public boolean isErrorIndicated() {
-        if (getError() == null)
+        if (getError() == null && getErrorDetail() == null)
```

Comment:
are you sure that error can't exist without error details? Shouldn't be it ORed?